docs/integrations/pack-docs.md

```diff
@@ -55,3 +73,195 @@ It is possible to add an image placeholder which links to an external video. For
+  - At the content of the Slack pack, locate the reason/s for the dependency. We find that the playbook `Slack - General Failed Logins v2.1` uses `ad-expire-password`:
+  
+![image](https://user-images.githubusercontent.com/43602124/89119604-51394480-d4b8-11ea-9a88-e8ce3889a48a.png)
+  - Solve the issue. In this case - we can add a condition before this task - “Is Active Directory enabled?”. Then, if the Active_Directory_Query pack is not present, the condition will lead to “else” and we just solved the required dependency issue.
+Change “mandatory” to “false” in Slack’s dependency on Active_Directory_Query.
+
+- CortexXDR pack depends on PortScan pack:
+
+![image](https://user-images.githubusercontent.com/43602124/89119624-70d06d00-d4b8-11ea-8021-c1e990039dff.png)
+  - Looking through the content of CortexXDR pack we find that `Cortex XDR Port Scan` incident type is configured to run `Port Scan - Generic` from the PortScan pack.
+  
+![image](https://user-images.githubusercontent.com/43602124/89119636-86de2d80-d4b8-11ea-856a-b358aef1c654.png)
+This one is obviously a bug, because the right playbook that should run is Cortex XDR - Port Scan, and not the generic port scan playbook.
+  - Change the playbook that the incident type is tied to
+  - Change mandatory to false.
+  
+- QRadar depends on AccessInvestigation pack
+  - We see that QRadar has a playbook called `Access Investigation - QRadar` which uses `Access Investigation - Generic`.
+  
+  ![image](https://user-images.githubusercontent.com/43602124/89119663-c7d64200-d4b8-11ea-8902-cd14f9682e5c.png)
+  - Access Investigation - Generic may have to be a Core pack, if so - its mandatory dependency will be fine. Note that packs become part of the Core packs only after our consideration and approval, and the list of core packs is meant to stay very limited. As a contributor, it is highly unlikely that your pack will be Core.
+Another solution may be to “skip if unavailable”. This is a feature that exists starting in XSOAR 5.5 where if the content is not available (E.g., pack is not installed), then the task and its continuous branch will be skipped:
+
+![image](https://user-images.githubusercontent.com/43602124/89119677-ee947880-d4b8-11ea-939b-2db8304c7786.png)
+
+Note: When should you skip and when should you “check if X is available”? Well, when a playbook uses a task that is tied to a certain integration, it makes sense to check if that integration is enabled beforehand, because the skipping feature only works on 5.5+. However, when a subplaybook is used, you can’t really check whether that subplaybook is available without using the “skip” feature, so in that case it should be skipped using the new feature.
```

Comment:
Can you maybe add a small example/screenshot for both cases? The subplaybook is not very clear.

Reply:
It's right above :) 
![image](https://user-images.githubusercontent.com/43602124/91474339-c410c200-e8a2-11ea-9d92-d2468013dc03.png)

Reply:
What I meant is, can you also add a screenshot of the "Check if X is available" so they are both shown one next to the other?